js/app/coopcycle-frontend-js/lastmile/redux.js

```diff
@@ -0,0 +1,5 @@
+// published version
+export * from 'coopcycle-frontend-js/lastmile/redux'
```

Comment:
Actually `dispatch` or `logistics` would have been more accurate but that's fine 🙂